lib/block-patterns.php

```diff
@@ -158,3 +158,62 @@
 							<!-- /wp:social-links -->',
 	)
 );
+
+add_action(
+	'init',
+	function() {
+
+		$core_block_patterns = array(
+			'text-two-columns',
+			'two-buttons',
+			'two-images',
+			'text-two-columns-with-images',
+			'text-three-columns-buttons',
+			'large-header',
+			'large-header-button',
+			'three-buttons',
+			'heading-paragraph',
+			'quote',
```

Comment:
Did you mean to unregister all the core patterns? I still see two showing up: 

<img width="355" alt="Screen Shot 2021-04-13 at 2 37 09 PM" src="https://user-images.githubusercontent.com/5375500/114603629-d0d50100-9c4c-11eb-9b1c-56391148075f.png">

Reply:
Those are new [as of this morning](https://github.com/WordPress/gutenberg/pull/30469), so they can stay in. 👍